### ISSUE TITLE
Fix 'test_providers/test_providers_update_from_year' tests in 4.2

### ISSUE THREAD

**Issue by damarisg** [wazuh/wazuh-qa#1563]
### Issue information

| Related issue |
|--|
| Closes https://github.com/wazuh/wazuh-qa/issues/1531|

When it comes to starting with the test fix, there is some information that can help you:
```
Module: Vulnerability Detector
Support Team:  "binary beasts"
Target: Manager   
OS: Linux
```
I add some information obtained while researched the logs that failed.

## Case:

|  Type | Description  |
|--|--|
|Test Path | test_providers/test_providers_update_from_year
| Consistent | yes
| Test Execution | 3/3 executions failed
| Cases Fails | 14
| Summary | It should to do research with more details because in the logs doesn't show any details.
|| There are some failed could be a problem with parser error or because it doesn't detect log.
| Error message | " Could not find the log event: .*Downloading .*cve.json\?after=2020-01-01.* "

In order to finish this issue the following tasks should be fulfilled:
- [ ] Research of fails.
- [ ] Apply Fix 
- [ ] Full Green/ Full Yellow  in `test_providers/test_providers_update_from_year` for 3 times. 
- [ ] Documentation of any important change done for these tests or the used tools.

**Comment by pereyra-m** [comment]
Results for `test_providers/test_providers_update_from_year`

| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[centos_test_update_from_year_1.log](https://github.com/wazuh/wazuh-qa/files/6812270/centos_test_update_from_year_1.log)| 2021-07-13| Matias | :red_circle:
| [centos_test_update_from_year_2.log](https://github.com/wazuh/wazuh-qa/files/6812269/centos_test_update_from_year_2.log)| 2021-07-13| Matias | :red_circle:
|[centos_test_update_from_year_3.log](https://github.com/wazuh/wazuh-qa/files/6812268/centos_test_update_from_year_3.log) | 2021-07-13| Matias | :red_circle:

**Comment by mdengra** [comment]
### 2021-07-15

Test results with the modules `sca`, `syscollector` and `rootcheck` modules disabled in the `ossec.conf`: 

| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[test_providers_update_from_year_local_dm_r1.log](https://github.com/wazuh/wazuh-qa/files/6822942/test_providers_update_from_year_local_dm_r1.log)| 2021-07-15 | Miguel | :green_circle: 
|[test_providers_update_from_year_local_dm_r2.log](https://github.com/wazuh/wazuh-qa/files/6822945/test_providers_update_from_year_local_dm_r2.log)| 2021-07-15 | Miguel | :green_circle: 
|[test_providers_update_from_year_local_dm_r3.log](https://github.com/wazuh/wazuh-qa/files/6822948/test_providers_update_from_year_local_dm_r3.log)| 2021-07-15 | Miguel | :green_circle:

**Comment by mdengra** [comment]
### 2021-07-20

Used Wazuh-QA branch: `1531-full-yellow-vuln-det`
Test results with the modules `sca`, `syscollector` and `rootcheck` modules disabled in the `ossec.conf`: 

| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[test_providers_update_from_year_local_dm_r1.log](https://github.com/wazuh/wazuh-qa/files/6846699/test_providers_update_from_year_local_dm_r1.log)| 2021-07-20 | Miguel | :yellow_circle: 
|[test_providers_update_from_year_local_dm_r2.log](https://github.com/wazuh/wazuh-qa/files/6846700/test_providers_update_from_year_local_dm_r2.log)| 2021-07-20 | Miguel | :yellow_circle: 
|[test_providers_update_from_year_local_dm_r3.log](https://github.com/wazuh/wazuh-qa/files/6846702/test_providers_update_from_year_local_dm_r3.log)| 2021-07-20 | Miguel | :yellow_circle:

**Comment by mdengra** [comment]
### 2021-07-23

Used Wazuh-QA branch: `1531-full-yellow-vuln-det`
Test results with the default settings in the `ossec.conf`: 

| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[test_providers_update_from_year_local_r1.log](https://github.com/wazuh/wazuh-qa/files/6867540/test_providers_update_from_year_local_r1.log)| 2021-07-23 | Miguel | :yellow_circle: 
|[test_providers_update_from_year_local_r2.log](https://github.com/wazuh/wazuh-qa/files/6867541/test_providers_update_from_year_local_r2.log)| 2021-07-23 | Miguel | :yellow_circle: 
|[test_providers_update_from_year_local_r3.log](https://github.com/wazuh/wazuh-qa/files/6867543/test_providers_update_from_year_local_r3.log)| 2021-07-23 | Miguel | :yellow_circle:

**Comment by damarisg** [comment]
We can detect that the tests sometimes fail, but when we go to the `ossec.log` there are the logs expected. It happens by the #1602 issue.

| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[test_providers_update_from_year_local_r1.log](https://github.com/wazuh/wazuh-qa/files/6867540/test_providers_update_from_year_local_r1.log)| 2021-07-23 | Seyla| 🔴  
|[test_providers_update_from_year_local_r2.log](https://github.com/wazuh/wazuh-qa/files/6867541/test_providers_update_from_year_local_r2.log)| 2021-07-23 |  Seyla| 🔴  
|[test_providers_update_from_year_local_r3.log](https://github.com/wazuh/wazuh-qa/files/6867543/test_providers_update_from_year_local_r3.log)| 2021-07-23 |  Seyla| 🔴

**Comment by damarisg** [comment]
This issue is closed because the test was disabled by #1602.
